### PR TITLE
[PW-4562] add help text to configuration

### PR DESCRIPTION
**Pull request by peterojo**
Add more helpful text to the available fields in the plugin configuration
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

**Review by cyattilakiss — APPROVED** [review]

File: src/Resources/config/config.xml [hunk 1]
```diff
@@ -7,29 +7,47 @@
         <input-field type="text">
             <name>merchantAccount</name>
             <label>Merchant account</label>
+            <helpText>
+                In your Adyen Customer Area you have a company account and under that a merchant account.
```
Comment (acampos1916) [inline src/Resources/config/config.xml:11]: Not sure about this wording. What do you think @Suzell

File: src/Resources/config/config.xml [hunk 2]
```diff
@@ -39,25 +57,56 @@
...
+                See https://docs.adyen.com/plugins/shopware-6#set-up-notifications.
+                For testing the plugin without notifications, this field can be empty.
+            </helpText>
         </input-field>
         <input-field type="password">
             <name>hmacLive</name>
             <label>HMAC Key for live</label>
+            <helpText>
+                This is used to authenticate your endpoints.
+                Fill in your live Hmac key from your live Adyen Customer Area &gt; Account &gt; Webhooks &gt; Edit or Add.
```
Comment (acampos1916) [inline src/Resources/config/config.xml:106]:
```suggestion
                Fill in your live HMAC Key from your live Adyen Customer Area &gt; Account &gt; Webhooks &gt; Edit or Add.
```

File: src/Resources/config/config.xml [hunk 3]
```diff
@@ -39,25 +57,56 @@
...
+                See https://docs.adyen.com/plugins/shopware-6#set-up-notifications.
+                For testing the plugin without notifications, this field can be empty.
+            </helpText>
         </input-field>
         <input-field type="password">
             <name>hmacTest</name>
             <label>HMAC Key for test</label>
+            <helpText>
+                This is used to authenticate your endpoints.
+                If you want to test the webhook notifications then get your test Hmac key from your test Adyen Customer Area &gt; Account &gt; Webhooks &gt; Edit or Add.
```
Comment (acampos1916) [inline src/Resources/config/config.xml:96]:
```suggestion
                If you want to test the webhook notifications then get your test HMAC Key from your test Adyen Customer Area &gt; Account &gt; Webhooks &gt; Edit or Add.
```

File: src/Resources/config/config.xml [hunk 4]
```diff
@@ -39,25 +57,56 @@
...
+                See https://docs.adyen.com/plugins/shopware-6#set-up-notifications.
+                For testing the plugin without notifications, this field can be empty.
+            </helpText>
         </input-field>
         <input-field type="password">
             <name>notificationPassword</name>
             <label>Password for standard notifications</label>
+            <helpText>
+                This is the password for basic authentication of your live endpoints.
+                Fill in your from your live Adyen Customer Area &gt; Account &gt; Webhooks &gt; Edit or Add.
```
Comment (acampos1916) [inline src/Resources/config/config.xml:86]:
```suggestion
                Fill in from your live Adyen Customer Area &gt; Account &gt; Webhooks &gt; Edit or Add.
```

**Review by acampos1916 — APPROVED** [review]

**Review by cyattilakiss — APPROVED** [review]

**Pull request merged**